api/src/main/java/io/minio/PostPolicy.java

```diff
@@ -167,14 +167,6 @@ public String bucketName() {
   }
 
   /**
-   * Returns form data of this post policy.
-   */
-  public Map<String,String> formData(String accessKey, String secretKey)
```

Comment:
I am OK about backward incompatible.  You should increment major version in this PR.